site/zenodo_rdm/templates/semantic-ui/invenio_checks/requests/help_message.html

```diff
@@ -7,6 +7,12 @@
 under the terms of the MIT License; see LICENSE file for more details.
 #}
 
+{% if has_draft %}
+<div class="ui warning message">
+    <div class="header">Note: Draft in progress</div>
+    <p>The checks and feedback are based on the current draft of the record. Any recent edits may not yet appear in the published version.</p>
+</div>
```

Comment:
```suggestion
    <div class="header"><i class="attention icon"></i> Draft editing in progress</div>
    <p>You're viewing checks and feedback results for unpublished changes.</p>
</div>
```

Then maybe we could also add another line if we know there is also a published record? Something like:


```jinja
{% if is_published %} The published version available to users may differ from what's shown here.{% endif %}
```